tests/models/test_models_unet_2d.py

```diff
@@ -182,8 +184,9 @@ def test_output_pretrained(self):
         self.assertTrue(torch_all_close(output_slice, expected_output_slice, rtol=1e-3))
 
 
-class NCSNppModelTests(ModelTesterMixin, unittest.TestCase):
+class NCSNppModelTests(ModelTesterMixin, UNetTesterMixin, unittest.TestCase):
```

Comment:
Curiosity: 

What does it mean by `NCSNpp`?

Reply:
https://github.com/yang-song/score_sde_pytorch